app/assets/javascripts/app/components/on_scroll/on_scroll.js

```diff
@@ -22,7 +22,7 @@
 
         refreshElement(); // run the first time
         setupAugmentedScrollEvents(scrollableElement);
-        scrollableElement.on('$scroll', $$rAF.throttle(refreshElement));
+        scrollableElement.on('$scroll', $$rAF.debounce(refreshElement));
```

Comment:
`$$rAF.throttle` was named `$$rAF.debounce` in the angular material version that is being used.

This should be changed in the future to use underscore's `throttle` instead.